components/fxa-client/src/http_client.rs

```diff
@@ -143,6 +167,25 @@ impl FxAClient for Client {
         }))
     }
 
+    fn ecosystem_anon_id(
+        &self,
+        config: &Config,
+        access_token: &str,
+        ecosystem_anon_id: &str,
+    ) -> Result<()> {
+        let url = config.auth_url_path("v1/account/ecosystemAnonId")?;
```

Comment:
Please use `config.profile_url_path("v1/ecosystem_anon_id")` here, we need to write via the profile-server rather than the auth-server.